sample_metadata/parser/generic_parser.py

```
@@ -199,6 +205,10 @@ def parse_manifest(
             sample_id = self.get_sample_id(row)
             sample_map[sample_id].append(row)
 
+        if len(sample_map) == 0:
+            logger.error(f'No sample information read from the manifest')
```

I think this should raise an exception.